src/main/java/org/folio/circulation/rules/Drools.java

```diff
@@ -78,12 +76,15 @@ private KieSession createSession(MultiMap params, Location location) {
    * @return CirculationRuleMatch object with the name of the loan policy and rule conditions
    */
   public CirculationRuleMatch loanPolicy(MultiMap params, Location location) {
-    KieSession kieSession = createSession(params, location);
-    RuleEventListener ruleEventListener = new RuleEventListener();
+    final var match = new Match();
```

Comment:
Have you deliberately started using `final var` instead of `val`?

Reply:
yes, there is `var` keyword since java 11, so I trying to use it instead of lombok's `val`.